experiment/resultstore/convert.go

```diff
@@ -175,13 +175,18 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 		artifacts[i] = "gs://" + bucket + "/" + a
 	}
 
+	var total int
 	for _, a := range artifacts { // add started.json, etc to the invocation artifact list.
+		if total >= maxFiles {
+			continue
```

Comment:
Should we log somewhere when we hit this case?

Reply:
Done